relay-server/src/actors/processor.rs

```diff
@@ -1760,10 +1760,6 @@ impl EnvelopeProcessorService {
                 .map_err(ProcessingError::QuotasFailed)?
         });
 
-        if enforcement.event_metrics_active() {
-            state.extracted_metrics.clear();
```

Comment:
This was introduced in #1537, but it is now dead code. Metrics will always be extracted at this point, and as such `event_metrics_active` can never be true. 

This is not a problem, since the extracted metrics will be sent to the project cache, which will immediately drop them.